### ISSUE TITLE
EmitC missing shims to render ML models

### ISSUE THREAD

**Issue by hcindyl** [google/iree#7842]
EmitC failed to reconcile some VM shim calls, and then failed to compile ML models.

For example, the following calls are missing for the mnist.mlir in [iree/samples/models](https://github.com/google/iree/tree/main/iree/samples/models)

```bash
call_0riiirii_r_import
call_0riirii_r_import
call_0rririi_v_import
```
Other models such as mobilenet has other calls missing such as `call_0rriCiD_v_import` or more

If I add those calls explicitly in [iree/vm/shims_emitc.h](https://github.com/google/iree/tree/main/iree/vm/shim_emitc.h) the model can be built and the application runs fine.

Please see the example in https://github.com/hcindyl/iree/tree/mnist_static_c-test for detail.

**Comment by lundong** [comment]
Adding a few more missing calls:
call_0r_rrr_shim
call_0r_rrrrrrrr_shim
call_0r_rrrrrrrrrrrr_shim

**Comment by benvanik** [comment]
this is not good. we need to prioritize not using these shims - see https://github.com/google/iree/pull/7831#discussion_r765912479

**Comment by hcindyl** [comment]
FYI, `call_0r_rrrrrrrr_shim` is used by Mobilenet SSD.

**Comment by marbre** [comment]
Should be fixed with PR [#8057](https://github.com/google/iree/pull/8057). Please reopen if the issues persists.
